src/Plugin/ExtraField/Display/ListPageFilters.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_List_pages\Plugin\ExtraField\Display;
+
+use Drupal\Core\Entity\ContentEntityInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\extra_field\Plugin\ExtraFieldDisplayFormattedBase;
+use Drupal\oe_list_pages\ListBuilderInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Extra field displaying the list page filters.
+ *
+ * @ExtraFieldDisplay(
+ *   id = "oe_list_page_filters",
+ *   label = @Translation("List page filters"),
+ *   deriver = "Drupal\oe_list_pages\Plugin\ExtraField\Derivative\ListPageDeriver",
+ *   visible = true
```

Comment:
mmm, should this be visible by default? it's a component so let's not add them automatically no?